src/main/java/core/basesyntax/service/validator/ValidatorImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.validator;
+
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class ValidatorImpl implements Validator {
+    @Override
+    public void validate(List<String> list) {
+        for (String record : list) {
+            record = record.trim();
+            if (!Pattern.matches("\\w,\\w+,\\d+", record)) {
+                throw new RuntimeException("Wrong input format " + record);
+            }
+            int value = Integer.parseInt(record.split(",")[2]);
```

Comment:
regex, `,` and `2` are magic literals - let's move them to constants
also, can we use such a regex that checks that number is positive?